alma/lib/Validators/PaymentValidation.php

```diff
@@ -347,4 +360,28 @@ private function getCartTotals($cart, $customer)
 
         return $cartTotals;
     }
+
+    /**
+     * @param $paymentId
+     * @param $apiKey
+     * @param $signature
+     *
+     * @return true
+     *
+     * @throws PaymentValidationException
+     */
+    public function checkSignature($paymentId, $apiKey, $signature)
+    {
+        if (!$paymentId) {
```

Comment:
i think this test is necessary only for log, isHmacValidated test if params are string

Reply:
@Francois-Gomis Do you want I replace throw by log ?

Reply:
No, need, if an error occur, you have exception log. If no error no logs